lib/webapi/webhandler.go

```diff
@@ -634,6 +649,21 @@ func getWebConfigAuthSettings(cfg WebHandlerConfig) telewebui.WebConfigAuthSetti
 	return authSettings
 }
 
+func (h *WebHandler) disableUI() bool {
```

Comment:
This reads as an action but is, in fact, a query - I'd rename this to `disabledUI` or `isDisabled`